src/web/db/task_store_pymysql.py

```
@@ -71,6 +71,41 @@ def update_stage(self, task_id: str, stage_name: str, stage_data: Dict[str, Any]
+        }
+        if column_name not in allowed_cols:
+            logger.error(f"Illegal stage column: {column_name!r}")
+
+        now = _current_ts()
+        try:
+            sql = (
+                "UPDATE task_stages "
+                f"SET {column_name} = %s, updated_at = %s "
+                "WHERE stage_id = %s"
+            )
+            self.db.execute_query(
```

Pylint (4.0.1)

[error] Instance of 'StageStore' has no 'db' member

(E1101)